packages/sdk-react/src/contexts/TurnkeyContext.tsx

```diff
@@ -99,6 +123,31 @@ export const TurnkeyProvider: React.FC<TurnkeyProviderProps> = ({
           iframeElementId: TurnkeyAuthIframeElementId,
         });
         setAuthIframeClient(newAuthIframeClient);
+
+        // Optionally include import iframe
+        // Should allow specification (config) of container / element ID
```

Comment:
not sure if this ultimately belongs here, in a provider that wraps an entire application